packages/devtools_app/lib/src/profiler/cpu_profile_model.dart

```diff
@@ -495,6 +582,14 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
 
 @visibleForTesting
 int stackFrameIdCompare(String a, String b) {
+  // Order the root first.
+  if (a == CpuProfileData.rootId) {
```

Comment:
need to short circuit with
if (a == b) return 0;

Reply:
done